etc/telegraf/telegraf.toml.j2

```diff
@@ -335,3 +335,14 @@
 
 ####################################################################################
 {% endif %}
+
+{% if appdynamics_enabled %}
+# if AppDynamics is enabled this 'exec' output is activated.
+# The command (python script) reads the metrics from STDIN and transform it
+# to the appropriate form for the AppDynamics Machine Agent HTTP Listener payload.
+[[outputs.exec]]
+  command = ["python3", "/home/vcap/app/buildpack/telemetry/appdynamics_telegraf_output.py"]
```

Comment:
I would make this path configurable / compose it outside the template - it should be possible to compose it using some helper variables that are already available.

Reply:
@markbastiaans It is possible but the problem is that the 'BUILDPACK_DIR' variable is located in the file 'stage.py' https://github.com/mendix/cf-mendix-buildpack/blob/LM-754_telegraf_to_appDynamics/buildpack/stage.py#L21
but to pass this variable to the 'telegraf.toml.j2' we need this variable in the 'telegraf.py' in the function "update_config" https://github.com/mendix/cf-mendix-buildpack/blob/LM-754_telegraf_to_appDynamics/buildpack/telemetry/telegraf.py#L180
which is called from "start.py".

So it seems some refactoring are required to implement this. For example move BUILDPACK_DIR to 'util.py'.

Reply:
@markbastiaans Do you know the other way to do this?

Reply:
`update_config` is called in `start.py` - `BUILDPACK_DIR` should be available there, and you're free to change the method there.

Another alternative is the (slightly dirty) trick that's used in `runtime.py`:

```python
BASE_PATH = os.getcwd()
```

This should get you the buildpack directory in the buildpack start phase.

Reply:
Proceeded with second option for now. In the future I suppose we can do some refactoring and test it properly.